office-plugin/src/main/java/org/artofsolving/jodconverter/util/ConfigUtils.java

```diff
@@ -21,29 +22,51 @@ public static String getHomePath() {
         } else {
             String separator = File.separator;
             if (userDir.contains(MAIN_DIRECTORY_NAME)) {
-                userDir = userDir + separator + "src" + separator +  "main";
+                userDir = userDir + separator + "src" + separator + "main";
             } else {
                 userDir = userDir + separator + MAIN_DIRECTORY_NAME + separator + "src" + separator + "main";
             }
         }
         return userDir;
     }
 
+    // 获取环境变量，如果找不到则返回默认值
+    @SuppressWarnings("SameParameterValue")
+    private static String env(String key, String def) {
```

Comment:
```suggestion
    private static String envOrDefault(String key, String def) {
```
这里命名这样是不是更好点？？因为我看到下面使用的地方的时候，明明知道这个意思，还是多想了下

Reply:
可以，改了